### ISSUE TITLE
This is wrong

### ISSUE THREAD

**Issue by ScottRButler** [MicrosoftDocs/sql-docs#8681]
sudo add-apt-repository "$(wget -qO- https://packages.microsoft.com/config/ubuntu/20.04/mssql-server-2022.list)"

sudo apt-add-repository "$(wget -qO- https://packages.microsoft.com/config/ubuntu/20.04/mssql-server-2022.list)"

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 5634d572-0020-808b-693a-13addc8861d9
* Version Independent ID: 136bab7d-d4d1-1e82-ba89-be2020bc784b
* Content: [Ubuntu: Install SQL Server on Linux - SQL Server](https://learn.microsoft.com/en-us/sql/linux/quickstart-install-connect-ubuntu?view=sql-server-linux-ver16&preserve-view=true)
* Content Source: [docs/linux/quickstart-install-connect-ubuntu.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/linux/quickstart-install-connect-ubuntu.md)
* Service: **sql**
* Sub-service: **linux**
* GitHub Login: @VanMSFT
* Microsoft Alias: **vanto**

**Comment by VanMSFT** [comment]
Hi @ScottRButler - `add-apt-repository` is the correct command. 

https://manpages.ubuntu.com/manpages/trusty/man1/add-apt-repository.1.html

Can you elaborate on why you think the command is incorrect? Thanks.

**Comment by ScottRButler** [comment]
> Hi @ScottRButler - `add-apt-repository` is the correct command.
> 
> https://manpages.ubuntu.com/manpages/trusty/man1/add-apt-repository.1.html
> 
> Can you elaborate on why you think the command is incorrect? Thanks.

Oddly enough, when I tried it yesterday it didn't seem to work but apt-add-repository did. I just tried it again on the same machine and it worked fine. Apparently both versions work.
My bad. :-)

Although I did need to run this
apt install software-properties-common
before I could run apt-add-repository "$(wget -qO- https://packages.microsoft.com/config/ubuntu/20.04/mssql-server-2019.list)"

--Scott
